src/props.animations.js

```diff
@@ -100,19 +100,19 @@ export default {
     opacity: .5
   }
 }`,
-  "--animation-float": "float 3s var(--ease-in-out-3) infinite", 
+  "--animation-float": "float var(--float-duration, 1ms) var(--ease-in-out-3) infinite", 
```

Comment:
```suggestion
  "--animation-float": "float 3s var(--float-ease, steps(2)) infinite", 
```

got a `steps(2)` here, breaking the pattern!